### PR TITLE
New package: gscan2pdf

### PR DESCRIPTION
**Pull request by dominikh**
gscan2pdf and all of its dependencies,.

goocanvas1 has the `1` prefix because it's an old version and could be installed alongside with the current version (2.x).

### CODE REVIEW

**Comment by dominikh** [comment]
> srcpkgs/perl-Gtk2-ImageView/template:10: short_desc should be less than 72 chars

xlint bug, description is 71 characters long

> srcpkgs/perl-Try-Tiny/template:14: use $CPAN_SITE

can someone figure out the right URL? I couldn't find Try-Tiny via `/by-module/`

**Comment by pullmoll** [comment]
Perhaps use DOY's github.com releases?
`https://codeload.github.com/doy/try-tiny/tar.gz/Try-Tiny-${version}>Try-Tiny-${version}.tar.gz`

You can as well use `tar.xz` or `tar.bz2` AFAIK by adjusting the path fragment.

**Comment by dominikh** [comment]
I'm quite happy with fetching from CPAN, I just don't like the xlint warning ;)

File: srcpkgs/goocanvas1/template [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# Template file for 'goocanvas1'
+pkgname=goocanvas1
+version=1.0.0
+revision=1
+wrksrc=goocanvas-${version}
+build_style=gnu-configure
+maintainer="Dominik Honnef <dominik@honnef.co>"
+hostmakedepends="pkg-config"
+makedepends="gtk+-devel glib-devel cairo-devel gobject-introspection"
```
Comment (xtraeme) [inline srcpkgs/goocanvas1/template]: - `glib-devel` and `cairo-devel` are redundant, `gtk+-devel` already depends on them.
- `gobject-introspection` must be moved to `hostmakedepends` and you'd have to add a build option `gir` to enable introspection, and disable it for cross compilation.

File: srcpkgs/goocanvas1/template [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+license="LGPL-2"
+homepage="https://wiki.gnome.org/action/show/Projects/GooCanvas"
+short_desc="Canvas widget for GTK+ - Version 1"
+distfiles="${GNOME_SITE}/goocanvas/1.0/goocanvas-${version}.tar.bz2"
+checksum=1c072ef88567cad241fb4addee26e9bd96741b1503ff736d1c152fa6d865711e
+
+
+goocanvas1-devel_package() {
+	short_desc+=" - development files"
+	depends="goocanvas1>=${version}_${revision} gtk+-devel cairo-devel"
```
Comment (xtraeme) [inline srcpkgs/goocanvas1/template]: `depends="${sourcepkg}>=${version}_${revision} ${makedepends}"`

File: srcpkgs/gtkimageview/template [hunk 3]
```diff
@@ -0,0 +1,35 @@
+# Template file for 'gtkimageview'
+pkgname=gtkimageview
+version=1.6.4
+revision=1
+_githash=77abd2122c19d2eca21599404a2d3aece35081e2
+wrksrc="${pkgname}-${_githash}"
+build_style=gnu-configure
+maintainer="Dominik Honnef <dominik@honnef.co>"
+hostmakedepends="gnome-common pkg-config gtk-doc libtool"
+makedepends="gtk+-devel glib-devel"
```
Comment (xtraeme) [inline srcpkgs/gtkimageview/template]: - `glib-devel` is redundant

File: srcpkgs/gtkimageview/template [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+}
+
+pre_build() {
+	sed -e '/CFLAGS/s/-Werror //g' -i configure.in
+	sed '/DEPRECATED_FLAGS/d' -i configure.in
+}
+
+gtkimageview-devel_package() {
+	shortdesc+=" - development files"
+	depends="gtkimageview>=${version}_${revision} gtk+-devel"
```
Comment (xtraeme) [inline srcpkgs/gtkimageview/template]: - ditto

**Comment by dominikh** [comment]
> glib-devel and cairo-devel are redundant, gtk+-devel already depends on them

gtk+-devel depends on libglib-devel, not glib-devel. glib-devel has tools such as `glib-genmarshal` in it.

**Comment by xtraeme** [comment]
Right, but `glib--devel` in `makedepends` is wrong anyway, should be `hostmakedepends`.

**Comment by xtraeme** [comment]
Please rebase when you have a chance, hopefully the build should be ok this time with travis.

**Comment by xtraeme** [comment]
What's the state of this? some pkgs were merged manually, but seems to be incomplete.

**Comment by dominikh** [comment]
Sourceforge had problems the past few days. I'll finish the merge soon.

**Comment by dominikh** [comment]
Merged manually.

**Pull request closed without merging**